app/src/test/java/org/simple/clinic/deeplink/DeepLinkUpdateTest.kt

```diff
@@ -75,6 +77,24 @@ class DeepLinkUpdateTest {
         ))
   }
 
+  @Test
+  fun `if user is logged in and user does not have capabilities, then show user is not authorised to teleconsult log`(){
```

Comment:
The name of this test does not reflect the actual test setup, right? It says that the `user does not have capabilities` but in the test setup method we actually are adding capabilities.

I feel we need to add two different tests here even though the outcome is the same (showing the not allowed to log teleconsult dialog).

- One where we do not know if the user is allowed to log a teleconsult or not
- One where we know the user is not allowed to log a teleconsult

In addition, since this is a test for business logic, we should not be using implementation specific phrases like `user does not have capabilities` and should instead use the use-case specific phrases like `is not aware of recording teleconsult logs`.

Reply:
> * One where we do not know if the user is allowed to log a teleconsult or not

We'll have capabilities set as `no` by default, so the user is always aware whether he/she is allowed to log teleconsult or not, right? Not sure if this test is required.

Reply:
This is not that scenario, this is the scenario where we upgrade from a version of the app where there is no capabilities saved for the user in the local database and the refresh user call fails and the capabilities are never fetched and saved.